src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -457,6 +462,17 @@ impl Window {
         Ok(())
     }
 
+    pub fn request_user_attention(&self, request_type: Option<UserAttentionType>) {
+        if !self.windowing_features.xdg_activation() {
+            warn!("'request_user_attention' isn't supported.");
```

Comment:
I see no other warning that uses punctuation at the end in winit at the moment. So removing that probably makes sense. Backticks also seem to be used exclusively instead of single quotes.

```suggestion
            warn!("`request_user_attention` isn't supported");
```